help/destinations/catalog/advertising/magnite-streaming.md

```diff
@@ -122,23 +120,33 @@ Once the destination connection has been created, you can proceed to the audienc
 
 ### Map attributes and identities {#map}
 
-The next step is mapping source identifiers to the Magnite device_id identifier.
+The next step is mapping Source Field identifiers to the [!DNL Magnite] Device Type identifier of your choice, located in the Target Field.
 
-* You can add as many mappings as you need by selecting **[!UICONTROL Add new mapping]**.
+The current list of [!DNL Magnite] supported device types will be available via a dropdown list, by clicking the Target Field's target attribute button.
```

Comment:
"The current list of [!DNL Magnite] supported device types will be available via a dropdown list, by clicking the Target Field's target attribute button.".

There is no dropdown when you configure mapping. Please rephrase this part so it reflects the UI.